yt/utilities/parameter_file_storage.py

```diff
@@ -187,7 +187,7 @@ def _write_out(self):
     @parallel_simple_proxy
     def read_db(self):
         """ This will read the storage device from disk. """
-        f = open(self._get_db_name(), "rb")
+        f = open(self._get_db_name(), "r")
```

Comment:
Since you're modifying this code already, could you make sure that `f` is closed? Preferably with:

```
with open(self._get_db_name(), "r") as f:
    vals = csv.DictReader(f, _field_names)
```